src/worldcereal/openeo/preprocessing.py

```diff
@@ -269,10 +269,11 @@ def raw_datacube_DEM(
                 "https://stac.openeo.vito.be/collections/COPERNICUS30_DEM_SLOPE",
                 spatial_extent=spatial_extent,
                 bands=["Slope"],
-            )
-            .rename_labels(dimension="bands", target=["slope"])
-            .min_time()
+            ).rename_labels(dimension="bands", target=["slope"])
+            # .min_time()
         )
+        slope.metadata = slope.metadata.add_dimension("t", "2020-01-01", "temporal")
```

Comment:
and in that case, this should no longer be required.

Reply:
There is an issue with CDSE that doesn't come with FED

With FED there is a temporal dimension detected and min_time drops it. In CDSE there is also a temporal dimension, but the openeo client is somehow unsynchronized and thinks there isn't (while the backend has it), so this is a workaround to allow it

Reply:
I updated that part, maybe if you read it again it will make more sense

Reply:
@kvantricht which version of the OpenEO python client do you have? I have `0.29.0`